query/sorting.go

```diff
@@ -29,7 +30,10 @@ func (c SortCriteria) GoString() string {
 // See: https://github.com/infobloxopen/atlas-app-toolkit#sorting
 func ParseSorting(s string) (*Sorting, error) {
 	var sorting Sorting
-
+	sortingRegexp := regexp.MustCompile("^[A-Za-z0-9_,. ]+$")
```

Comment:
This change makes sense. I just would like to clarify why this set of allowed characters is choosen?
I think we should also allow `-` (maybe something else?)

Reply:
Updated the regexp pattern, added  dash.
As for other special characters, the following are allowed:
1. Any letters and numbers
2. Underscore as it can be used in the object name
3. Comma as a separator in the array of sorting criteria.
4. Dot that can be used for nested objects